index.html

```diff
@@ -52,7 +52,73 @@ <h3 class="stories">Multi-Post Stories</h3>
     <a href="#"><li class="tag3">Bootstrap</li></a>
     <a href="#"><li class="tag4">Ruby</li></a>
   </ul>
-  <div><a href="#" class="see__project">See Project</a></div>
+    <a href="#" class="see__project">See Project</a>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
+    </section>
+    <section class="section3">
+      <h4 class="section3-heading">Proffesional Art Printing Data</h4>
+      <p class="section3-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+      <ul class="section3-items">
+        <a href="#" class="section3-item1"><li>HTML</li></a>
+        <a href="#" class="section3-item2"><li>Bootstrap</li></a>
+        <a href="#" class="section3-item3"><li>Ruby</li></a>
+      </ul>
+      <a href="#" class="section3__project">See Project</a>
```

Comment:
- [ ] Good job using an `a` tag for your buttons but the button interactions have not yet been implemented. This is the design from the Figma template so it would be nice if you could add that.
![f5](https://user-images.githubusercontent.com/85113567/148490517-5887eee5-56e4-4d0f-ac72-5a929c03840e.jpg)